### PR TITLE
Use classgraph library to scan classpath contents

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: org.eclipse.xtend.core.tests/.classpath [hunk 1]
```diff
@@ -14,6 +14,7 @@
 	</classpathentry>
 	<classpathentry kind="src" output="bin/test" path="testdata">
 		<attributes>
+			<attribute name="ignore_optional_problems" value="true"/>
```
Comment (cdietrich) [inline org.eclipse.xtend.core.tests/.classpath]: @szarnekow is this a thing gradle adds or do we miss a config in gradle?

Comment (szarnekow) [reply org.eclipse.xtend.core.tests/.classpath]: I changed this in the UI. My bad.

Comment (cdietrich) [reply org.eclipse.xtend.core.tests/.classpath]: do you think we should add this to gradle for all (tests?) projects?

Comment (szarnekow) [reply org.eclipse.xtend.core.tests/.classpath]: It would make sense for the source folders, that do contain test-java-code

**Pull request merged**